src/changelog.ts

```diff
@@ -364,42 +351,35 @@ export default class Changelog {
 
   #formatter: Formatter;
 
-  readonly #versionBeforePackageRename: string | undefined;
-
-  readonly #tagPrefixBeforePackageRename: string | undefined;
+  readonly #packageRename: PackageRename | undefined;
 
   /**
    * Construct an empty changelog.
    *
    * @param options - Changelog options.
    * @param options.repoUrl - The GitHub repository URL for the current project.
-   * @param options.tagPrefix - The prefix used in tags before the version number.
-   * @param options.formatter - A function that formats the changelog string.
-   * @param options.versionBeforePackageRename - A version string of the package before being renamed.
-   * An optional, which is required only in case of package renamed.
-   * @param options.tagPrefixBeforePackageRename - A tag prefix string of the package before being renamed.
+   * @param [options.tagPrefix] - The prefix used in tags before the version number.
+   * @param [options.formatter] - A function that formats the changelog string.
+   * @param [options.packageRename] - The package rename properties.
```

Comment:
@MajorLift and I had a discussion about using brackets to indicate optional arguments [here](https://github.com/MetaMask/auto-changelog/pull/159), and we decided that they aren't necessary after all, since both VSCode and `typedoc` both ignore these, and since we already use TypeScript types to communicate optionality. For consistency with our libraries would you mind removing these? Sorry for the back and forth on this.

``` suggestion
   * @param options.tagPrefix - The prefix used in tags before the version number.
   * @param options.formatter - A function that formats the changelog string.
   * @param options.packageRename - The package rename properties.